### ISSUE TITLE
supporting training iteration control required

### ISSUE THREAD

**Issue by hyunghunny** [hyunghunny/tf-hpo#1]
For the CNN layer 2 + fully connected layer, followings are the fixed parameters:
    # Parameters
    learning_rate = 0.001
    training_iters = 200000
    batch_size = 128
    display_step = 10  
    dropout = 0.75 # Dropout, probability to keep units

To minimize the training time, reducing training_iters will be required first.
